extensions/medical/handleDamage.cpp

```diff
@@ -54,11 +54,12 @@ namespace ace {
             return stream.str();
         }
 
-        std::vector<ace::medical::injuries::OpenWound> handleDamage::GetInjuryInfoFor(const std::string& typeOfDamage, double amountOfDamage, int selection, int woundID)
+        std::vector<ace::medical::injuries::OpenWound> handleDamage::GetInjuryInfoFor(const std::string& typeOfDamage, double amountOfDamage, std::string& selectionName, int woundID)
```

Comment:
amountOfDamage and woundID are also const values

Reply:
selectionName should be const to not accidentally modify it in here.